packages/rum/src/boot/recorderApi.ts

```diff
@@ -53,7 +53,7 @@ export function makeRecorderApi(
   startRecordingImpl: StartRecording,
   createDeflateWorkerImpl?: CreateDeflateWorker
 ): RecorderApi {
-  if (canUseEventBridge() || !isBrowserSupported()) {
+  if (!isBrowserSupported()) {
```

Comment:
**❓ question:** ‏Just to confirm, outdated versions of Mobile SDKs bridges will behave correctly when sending records, right? They won't send them as RUM events?

Reply:
Very good question! Marius told me it should be good. But we have to do proper tests (also for IOS). I did not add a feature flag because we should be able to test it soon. But I won't merge before testing ;)

Reply:
I've added a feature detection to only send records when the bridge supports it. https://github.com/DataDog/browser-sdk/pull/2470/commits/e05cc186cc50ffebcd1c51c00ca1c45180b254f0
Let me know what you think